src/calculateRentalCost.js

```diff
@@ -5,6 +5,16 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Replace the literal '50' with a named constant to clearly represent the long term discount.